service.watchedlist/resources/language/Hebrew/strings.po

```diff
@@ -0,0 +1,410 @@
+# XBMC Media Center language file
```

Comment:
please adjust to the new language folder layout /resources/language/resource.language.he_il/strings.po